### ISSUE TITLE
memcached-tool dump: infinite expiration time is not dumped as 0

### ISSUE THREAD

**Issue by ronald** [memcached/memcached#335]
I dumped using `memcached-tool dump` and restored using `nc`.
All items without expiration time (value 0) are missing.

```$ echo -e 'add my_key 0 0 11\r\nhello world\r' | nc localhost 11211
STORED
$ /usr/share/memcached/scripts/memcached-tool localhost:11211 dump 
Dumping memcache contents
  Number of buckets: 1
  Number of items  : 1
Dumping bucket 1 - 1 total items
add my_key 0 1518017432 11
hello world
```

expected result 
```add my_key 0 0 11```

The article https://blog.elijaa.org/2010/12/24/understanding-memcached-stats-cachedump-command/ indicates that the timestamp must be combined with uptime & current time from memcached stats but there is still a difference of 60 seconds.

**Comment by dormando** [comment]
There was a PR someone had last year trying to fix this :(

The tool should really use the new lru_crawler metadump command, which is able to print 0 exp time values correctly (displays them as -1).

If you're curious, you can telnet in from a recent version and run: `lru_crawler metadump 1` and it'll dump everything in slab class 1.

I'm going to leave this open for a bit.. maybe I should patch that or add a wiki document.

**Comment by calin-iorgulescu** [comment]
I took a crack at updating `memcached-tool`'s `dump` command to use `lru_crawler`. However, I simply pull all key information using `lru_crawler metadump all`, instead of pulling each the keys for each slab separately.

Please see #361  .

**Comment by dormando** [comment]
released in 1.5.7
